src/nvme/ioctl.c

```diff
@@ -1927,8 +1927,8 @@ int nvme_resv_report(int fd, __u32 nsid, bool eds, __u32 len,
 }
 
 int nvme_zns_mgmt_send(int fd, __u32 nsid, __u64 slba, bool select_all,
-		       enum nvme_zns_send_action zsa, __u32 data_len,
-		       void *data)
+		       __u32 timeout, enum nvme_zns_send_action zsa,
+		       __u32 data_len, void *data)
```

Comment:
Odd. Why do have zns command a timeout, and other commands haven't?
I'm pretty sure we can set a timeout on any command, so what makes these special?

Reply:
I don't know, this is from  fa97500b9a72 ("Add Timeout Parameter for Zone Management Commands")
https://github.com/linux-nvme/nvme-cli/commit/fa97500b9a72

Reply:
I suppose we should add the timeout to all commands then. I'll created an issue to track this, see #97